helm_deploy/values-prod.yaml

```diff
@@ -4,13 +4,20 @@
 generic-service:
   ingress:
     host: electronic-monitoring-datastore-api.hmpps.service.justice.gov.uk
-    # TODO: Create certificate as per https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/custom-domain-cert.html
 
   env:
-    HMPPS_AUTH_URL: "https://sign-in.hmpps.service.justice.gov.uk/auth"
-    # Template kotlin calls out to itself to provide an example of a service call
-    # TODO: This should be replaced by a call to a different service, or removed
-    # EXAMPLE_API_URL: "https://electronic-monitoring-datastore-api.hmpps.service.justice.gov.uk"
+    SPRING_PROFILES_ACTIVE: dev
```

Comment:
Why are we setting the `dev` spring profile to be active in production deployment?

Similarly, the Auth URL below should surely be the prod one?

Same questions apply for [values-preprod](helm_deploy/values-preprod.yaml).